src/services/newMessagesService.ts

```diff
@@ -98,22 +101,34 @@ export default class NewMessagesService {
 
       return withValidatedOrInternalError(resMessageContent, async response => {
         if (response.status === 200) {
-          const maybePrescriptionData = fromNullable(
-            response.value.content.prescription_data
+          const responseValue = response.value;
+          const errorOrMessageWithContent = CreatedMessageWithContent.decode(
+            responseValue
           );
```

Comment:
It's true that `CreatedMessageWithContent` is not an exact type, but we're stripping away `EnrichedMessage` property from static type.